lib/abci/handlers/deliverTxHandlerFactory.js

```diff
@@ -29,6 +40,10 @@ function deliverTxHandlerFactory(
    * @return {Promise<abci.ResponseDeliverTx>}
    */
   async function deliverTxHandler({ tx: stateTransitionByteArray }) {
+    const stHex = Buffer.from(stateTransitionByteArray).toString('hex');
+
+    logger.debug(`Delivering state transition ${stHex}`);
```

Comment:
Oh.. It will be too big. I guess we need a hash of ST as an identifier. I would add a block height as well.
```
Deliver state transition ${stHash} from block #${height}`
```